src/components/Avatar/Avatar.vue

```diff
@@ -0,0 +1,183 @@
+<!--
+  - @copyright Copyright (c) 2018 Julius Härtl <jus@bitgrid.net>
+  -
+  - @author Julius Härtl <jus@bitgrid.net>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<!-- TODO: Get rid of the inline css -->
+	<div v-tooltip="displayName" v-click-outside="closeMenu"
+		:class="{ 'icon-loading': loading, 'unknown': unknown }"
+		:style="{width: size + 'px', height: size + 'px'}"
```

Comment:
Ideally we should not have inline styles here, but we would loose the flexibility so specify any avatar size.

Reply:
IMO totally valid in this case :+1:

Reply:
Yes, just in case we might want to harden the server CSP to disallow inline script, but let's keep it for now.